packages/bson-transpilers/codegeneration/CodeGenerationVisitor.js

```diff
@@ -107,18 +107,16 @@ module.exports = (ANTLRVisitor) => class CodeGenerationVisitor extends ANTLRVisi
         }
       });
     this.requiredImports.driver = !!driverSyntax;
-    const imports = Object.keys(this.requiredImports)
-      .filter((code) => {
-        return (
-          this.requiredImports[code] &&
-          this.Imports[code] &&
-          this.Imports[code].template
-        );
-      })
-      .reduce((obj, c) => {
-        obj[c] = this.Imports[c].template(this.requiredImports[c]);
-        return obj;
-      }, {});
+    const imports = {};
+    for (const code in this.requiredImports) {
```

Comment:
tiny nit, but it’s a good practice to avoid `for ... in` unless it’s intentional to include enumerable keys from an object’s prototype chain (i.e. what the `Object.prototype.hasOwnProperty.call()` call would have tested for as well):

```suggestion
    for (const code of Object.keys(this.requiredImports)) {
```